client/ayon_core/hosts/houdini/startup/OPmenu.xml

```diff
@@ -0,0 +1,543 @@
+    It's used to extend the OPMenu while keeping the order of the menu items.
+-->
+
+
+<!--
+    This file defines the menu items when right-clicking over nodes in
+    Houdini.
+
+    Files named the same (ie. OPmenu.xml) can exist in $HOME/houdiniX.Y/
+    directory and will supplement the original definitions found in
+    $HFS/houdini/ (aka $HH).
+
+    The format in this file is similar to the format in $HH/MainMenuMaster
+    except that it uses the <menuDocument> and <menu> tags instead of
+    <mainMenu> and <menuBar>. See the comments in that file for more advanced
+    customization abilities.
+
+    Note that the id's correspond to their hotkeys with the h.pane.wsheet.
+    prefix (where available). For example, the item with id,
+    "opmenu.flag_display", refers to the action in the hotkeys manager as
+    "h.pane.wsheet.opmenu.flag_display".
+
+    Also note that you can specify which context a particular menu item shows
+    up in. To do this, add a context property:
+        <context>
+            <networkCategory>Object</networkCategory>
+        </context>
+    Or, for multiple contexts, a comma-separated list:
+        <context>
+            <networkCategory>Object,Sop,Dop</networkCategory>
+        </context>
+    Or use an expression to determine if the menu item appears in the current
+    context.  For example, to make the menu item appear only if the node is an
+    Object node:
+        <context>
+            <expression>kwargs["node"].type().category().name() == "Object"</expression>
+        </context>
+
+    For context expressions and menu scripts, a 'kwargs' python dictionary is
+    provided with the following values:
+        'node'      :   The node that the menu was opened for.
+        'toolname'  :   The menu item's hotkey id (ie. the element's id
+                        prefixed with h.pane.parms.)
+        'altclick'  :   True if the Alt key was also pressed, False otherwise
+        'ctrlclick' :   True if the Ctrl key was also pressed, False otherwise
+        'shiftclick':   True if the Shift key was also pressed, False otherwise
+        'cmdclick'  :   True if the Option key was also pressed, False
+                        otherwise. Note that this can only be true on OSX.
+-->
+
+<menuDocument>
+    <!-- menuDocument can only contain 1 menu element, whose id is
+         implicitly "root_menu"
+      -->
+    <menu>
+
+        <!-- This item appears at the top of the menu when you right-click an
+        "example" subnet -->
+
+        <!--
+        <scriptItem>
+            <label>Save Node Example...</label>
+            <context>
+                <expression>kwargs['node'].userData("__example_for") is not None</expression>
+            </context>
+            <scriptCode><![CDATA[
+from houdinihelp import examples
+examples.save_node_example(kwargs['node'])
+]]></scriptCode>
+        </scriptItem>
+        -->
+
+        <!-- Node flags submenu. -->
+        <subMenu id="opmenu.flags_menu">
```

Comment:
where is all of these coming from?

Reply:
correct

Reply:
as explained here https://www.sidefx.com/docs/houdini/basics/config_menus.html#reorganizing-menus

Reply:
oh, I think I missed! 
Let me try again.

Reply:
Please let me know what do you think
https://github.com/ynput/ayon-core/pull/504/commits/86b2f3b5b52431f6cb11b91e771552c717fa6df8

Reply:
much better yeah, ty!

Reply:
definitely much better!